src/crimson/os/seastore/cached_extent.h

```diff
@@ -553,7 +553,8 @@ class ExtentIndex {
   template <typename T>
   void remove(T &l) {
     for (auto &ext : l) {
-      extent_index.erase(l);
+      static_assert(std::is_same_v<decltype(ext), CachedExtent::index::value_type>);
+      extent_index.erase(ext);
```

Comment:
@athanatos i'd suggest just drop this method. what do you think?

Reply:
Yeah, if it's unused just remove it.